requirements.txt

```diff
@@ -7,6 +7,7 @@ psutil>=5
 scikit-learn>=0.20.0
 dlx
 docplex
+cplex; python_version < '3.8'
```

Comment:
As discussed in the meeting today, we probably shouldn't add cplex in the requirements but instead do a check if it is installed and then enable the optimization stack facilities. Could you remove this from the requirements?